content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Mobile Session Replay Setup and Configuration
+kind: documentation
+description: Describes how to setup and configure Mobile Session Replay
+aliases:
+further_reading:
+    - link: '/real_user_monitoring/session_replay/mobile'
+      tag: Documentation
+      text: Mobile Session Replay
+    - link: '/real_user_monitoring/session_replay/mobile/app_performance'
+      tag: Documentation
+      text: How Mobile Session Replay Impacts App Performance
+    - link: '/real_user_monitoring/session_replay/mobile/privacy_options'
+      tag: Documentation
+      text: Mobile Session Replay Privacy Options
+    - link: '/real_user_monitoring/session_replay/mobile/troubleshooting'
+      tag: Documentation
+      text: Troubleshoot Mobile Session Replay
+    - link: '/real_user_monitoring/session_replay'
+      tag: Documentation
+      text: Session Replay
+---
+
+## Setup
+
+{{< tabs >}}
+{{% tab "Android" %}}
+
+All Session Replay SDK versions can be found in the [Maven Snapshots Repository][1].
+
+To set up Mobile Session Replay for Android:
+
+1. Make sure you've [setup and initialized the Datadog Android RUM SDK][2].
+2. In order to use the SNAPSHOTS in your Gradle dependencies, add this repository as a source in your repository configuration in your `build.gradle` file:
+
+   ```kotlin
+     repositories {
+          maven {
+           url ="https://oss.sonatype.org/content/repositories/snapshots"
+          }
+          }
+
+          dependencies {
+             implementation 'com.datadoghq:dd-sdk-android:1.19.0-sr-beta1-SNAPSHOT'
+             implementation 'com.datadoghq:dd-sdk-android-session-replay:1.19.0-sr-beta1-SNAPSHOT'
+             implementation 'com.datadoghq:dd-sdk-android-session-replay-material:1.19.0-sr-beta1-SNAPSHOT' // in case you need material support
+
+          }
+   ```
+
+3. Once the DD SDK and Session Replay SDK dependencies are imported, you can enable the feature when configuring the SDK:
+   ```kotlin
+      val config = Configuration.Builder(
```

Comment:
we need to remove this part as this is related with the old way of initializing the SDK. For sdkv2 we have the new docs that will be available by Dash.

Reply:
removed

Reply:
can you update this section to reflect the new way? and in any where that is applicable, link to the new docs?

Reply:
I am not sure it will be required 🤔 . By the time people will reach to this docs the new docs about the SDK v2 initialization should be in place. Meaning that users will have to go first through those, initialize the SDK and then initialize the new SR feature. I will check with the team and maybe provide the link to new docs here.